aries-site/src/examples/foundation/content-container-sizes/BorderWidths.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Box, FormField, Select } from 'grommet';
+import { useDesignTokens } from '../../../components/content/designTokenUtils';
+
+export const BorderWidths = () => {
+  const [width, setWidth] = React.useState('default');
+  const { data } = useDesignTokens('semantic.borderWidth');
+
+  const containerTokens = data.map(
+    token => token.token.split('hpe.borderWidth.')[1],
+  );
+
+  return (
+    <Box direction="row" justify="between">
+      <Box
+        pad={{ vertical: 'xlarge', horizontal: 'xlarge' }}
+        align="center"
+        justify="center"
+        flex
+      >
+        <Box
+          height="xxsmall"
+          width="xxsmall"
+          border={{ color: 'decorative-blue', style: 'solid', size: width }}
```

Comment:
when the `width` size is "none" we get this, should we add some text or something? that the border goes away I know it also might be obvious but just thinking out loud
<img width="1266" height="319" alt="Screenshot 2025-12-03 at 12 21 32 PM" src="https://github.com/user-attachments/assets/94c6a986-acfa-4caf-8078-a4fc2ba1faaf" />

Reply:
Hmmm.... Good question. I'm open to others thoughts here, but since it defaults to a selection with a value, it has surrounding context of "border widths" indicating what is being affected, the user is making the choice, and the choice is "none," I'm not sure that is necessary.

Reply:
yes agree just was thinking out loud Im fine if we leave as is